developer/src/kmcmplib/src/CheckFilenameConsistency.cpp

```diff
@@ -91,16 +93,16 @@ KMX_DWORD CheckFilenameConsistency(KMX_WCHAR const * Filename, bool ReportMissin
   //  std::wstring -> wchar_t*
   const KMX_WCHART* wchptr = wstr.c_str();
 
+  nfile = _wfsopen(wchptr, L"rb", _SH_DENYWR);
   _wfinddata_t fi;
   n = _wfindfirst(wchptr, &fi);
   _findclose(n);
-#else
```

Comment:
I'm wondering if it would be good to output either a compiler warning when trying to build on non-Windows, or a message when trying to run.